### PR TITLE
List decompositions for torch.export

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py [hunk 1]
```diff
@@ -101,16 +103,197 @@ def scaled_dot_product_flash_attention(
...
 
 def get_inf_decomposition_list():
-    return ([torch.ops.aten.nll_loss_forward.default])
+    return [torch.ops.aten.nll_loss_forward.default]
+
+
+def get_export_decomposition_list():
+    # List of decompositions from torch._decomp.core_aten_decompositions
+    # removed _backward ops and ops supported without decomposition
+    decomp = [
```
Comment (rkazants) [inline src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py:127]: How are we going to maintain this list from one version to another?
Should we really need it for models conversion or is it for NNCF tool only?

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py:127]: This list changes not very often, we do not really need to always have it up to date. We may even reduce it in future. The main reason this list is needed is that SDPA gets decomposed by default but we want it to exist as a single operation.

Comment (rkazants) [reply src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py:127]: Can we check in the test that SDPA op is really created in the original graph?

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/decompositions.py:127]: The decompositions happen inside `convert_model`. Checking that operation exist would require patching ovc, also would need to decide how to split ts ops from fx ops when writing a test. This solution is outside of scope of this PR. I suggest solving that with a different PR

**Review by eaidova — APPROVED** [review]

**Review by daniil-lyakhov — APPROVED** [review]
vit_b_16 torchvision model now has SDPA node
image

**Pull request merged**